storage/namespace.go

```diff
@@ -728,6 +732,51 @@ func (n *dbNamespace) Flush(
 	return res
 }
 
+func (n *dbNamespace) Snapshot(blockStart, callStart time.Time, flush persist.Flush) error {
```

Comment:
nit: s/callStart/SnapshotTime/g

Reply:
yeah thanks, also had the callStart n.nowFn() issue which I fixed